src/storage/irc_catalog.cpp

```diff
@@ -137,13 +137,13 @@ DatabaseSize IRCatalog::GetDatabaseSize(ClientContext &context) {
 
 IRCEndpointBuilder IRCatalog::GetBaseUrl() const {
 	auto base_url = IRCEndpointBuilder();
-	base_url.SetPrefix(prefix);
```

Comment:
Does Polaris not need the prefix when querying the catalog? I know for hitting the config endpoint, the prefix is not needed, but the `GetConfig` function handles this. 

I can imagine a scenario where the prefix is an empty string "", which may break the endpoint builder, but then you can handle that in `IRCEndpointBuilder::GetUrl()`

Reply:
No it does not, the warehouse is the same thing so it gets included twice and leads to an error

Reply:
```c++
IRCEndpointBuilder IRCatalog::GetBaseUrl() const {
	auto base_url = IRCEndpointBuilder();
	base_url.SetVersion(version);
	base_url.SetHost(host);
	switch (catalog_type) {
	case ICEBERG_CATALOG_TYPE::AWS_GLUE:
	case ICEBERG_CATALOG_TYPE::AWS_S3TABLES: {
		base_url.AddPathComponent("iceberg");
		base_url.AddPathComponent(version);
		break;
	}
	default:
		break;
	}
	return base_url;
}
```

You should fetch main and merge with it 👍